### DISCUSSION TITLE
Displayed Precision

### DISCUSSION THREAD

**Discussion by CoveCompound** [dolezsa/thermal_comfort#178] [General]
My variables are being displayed to 2 decimal points. Is it possible to change the displayed precision to be one decimal point?

**Comment by rautesamtr** [comment]
Currently no, you can follow #196 but it currently isn't a priority

**Comment by BlackiceSpain** [comment]
> My variables are being displayed to 2 decimal points. Is it possible to change the displayed precision to be one decimal point?

you can do it in the options of the variable: 
![image](https://github.com/user-attachments/assets/e6839a0d-2ca3-4cf6-acd2-44c2966c287a)

**Reply by CoveCompound** [reply]
That request was from 3 years ago. I gave up on this long ago.
